### ISSUE TITLE
cyber timer bucket error?

### ISSUE THREAD

**Issue by daohu527** [ApolloAuto/apollo#10872]
https://github.com/ApolloAuto/apollo/blob/318d1484a87d008f096ca73d33d80f4f6a97a505/cyber/timer/timing_wheel.cc#L82
I found that when the "assistant_ticks" is 1, there's a issue.
Here is a sample:
```
current_work_wheel_index = 200
Timer.period = task->next_fire_duration_ms / TIMER_RESOLUTION_MS = 600
```
Then we get
work_wheel_index = 200 + 600 = 800
real_work_wheel_index = 800 & 511 = 288
assistant_ticks = 800 / 512 = 1

Then we will put "work_wheel_[real_work_wheel_index].AddTask(task)", which means put task into bucket 288, and current bucket is 200, so we will run task in 88 ticks. But we want the task run after 600 ticks.

I think maybe the condition change to:
```
if (assistant_ticks == 1 &&
        real_work_wheel_index < current_work_wheel_index_)
```
or just delete this condition, if the "work_wheel_index >= WORK_WHEEL_SIZE" then put in into the "assistant_wheel_"

**Comment by fengqikai1414** [comment]
Fixed with #10877

**Comment by daohu527** [comment]
Thank you for your quick feedback  : )
